routes/api-routes.js

```diff
@@ -19,7 +20,7 @@ module.exports = app => {
       user.findAll(req, res);
     }
   });
-  app.get("/api/user/:id", (req, res) => {
+  app.get("/api/user/:id", ensureAuthenticated, (req, res) => {
```

Comment:
I believe this route needs to be left unprotected so the usernames of other users can be displayed.

Reply:
Should be done.